include/spdk/bdev.h

```diff
@@ -190,6 +190,7 @@ enum spdk_bdev_io_status {
 	SPDK_BDEV_IO_STATUS_FAILED = -1,
 	SPDK_BDEV_IO_STATUS_PENDING = 0,
 	SPDK_BDEV_IO_STATUS_SUCCESS = 1,
+	SPDK_BDEV_IO_STATUS_NVME_ERROR = 2,
```

Comment:
I'm not sure if we still have any code that relies on it, but the error status should probably be negative (-2 would be fine).